app/routes/chatbot.js

```diff
@@ -86,33 +75,20 @@ router.post('/', (req, res) => {
     mobileCommonsProfileId: req.body.profile_id,
   });
 
-  let configured = true;
-  // Check for required config variables.
-  const settings = [
-    'GAMBIT_CMD_MEMBER_SUPPORT',
-    'GAMBIT_CMD_REPORTBACK',
-    'MOBILECOMMONS_OIP_AGENTVIEW',
-    'MOBILECOMMONS_OIP_CHATBOT',
-  ];
-  settings.forEach((configVar) => {
-    if (!process.env[configVar]) {
-      const msg = `undefined process.env.${configVar}`;
-      stathat(`error: ${msg}`);
-      logger.error(msg);
-      configured = false;
-    }
-  });
-
-  if (!configured) {
-    return res.sendStatus(500);
-  }
```

Comment:
would failing this check bring the whole app down? If not, it could be moved to its own middleware that would affect only a determined route. Otherwise I agree it should go somewhere in `server.js`.